UnitsNet/BaseDimensions.cs

```diff
@@ -41,6 +42,18 @@ public BaseDimensions(int length, int mass, int time, int current, int temperatu
             LuminousIntensity = luminousIntensity;
         }
 
+        public bool IsBase()
```

Comment:
`IsBaseUnit` and `IsDerivedUnit` maybe? I had to think a bit when reading usages to get exactly what `IsBase` meant.